services/movie.py

```diff
@@ -1,14 +1,19 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
+    title: str = None,
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
If argument can be None you have to show it in type hint. Fix it here and in other functions
```suggestion
    title: str | None = None,
    genres_ids: list[int] | None = None,
    actors_ids: list[int] | None = None,
```